subsys/mem_mgmt/Kconfig

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2023 Carlo Caione <ccaione@baylibre.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config MEM_ATTR
+	bool "Memory Attributes management library"
+	default y if ARM_MPU
```

Comment:
When the MPU is used we must enable this. The MPU drivers are using the mem attr library to build the regions mapping at run time instead of building that at compile time.

About the size increase I would not be concerned, we are moving the mappings from compile-time to run-time but yes, the library is slightly increasing the code size.